spec/system/admin/orders_spec.rb

```diff
@@ -922,3 +933,12 @@
     end
   end
 end
+
+def extract_pdf_content
+  pdf_href = page.all('a').pluck('href')
+  page.find(class: "button", text: "VIEW FILE").click
+  invoice_file_number = pdf_href[0][45..59]
+  invoice_path = "tmp/invoices/#{invoice_file_number}.pdf"
+  reader = PDF::Reader.new(invoice_path)
+  @invoice_content = reader.pages.map(&:text)
+end
```

Comment:
This block should be within the test description instead of being a global method. Otherwise it will leak into other specs.

Reply:
> This block should be within the test description

Ok, I've created a `context` for the test and moved the method it in there - I'm not 100% sure this is what you meant though?